Tests/SwiftGenKitTests/StringsTests.swift

```diff
@@ -75,4 +75,12 @@ class StringsTests: XCTestCase {
       XCTFail("Unexpected error occured while parsing: \(error)")
     }
   }
+
+  func testCustomSeparator() throws {
+    let parser = Strings.Parser(options: ["separator": "_"])
```

Comment:
I think it could be worth using a multi-character separator for this test, like `["separator": "__"]` (which is the use case that was requested in #576 btw)

Reply:
Rebased with some test keys in Localizable (some with `__` and some with the old `_`)